tests/validation/converters/test_vision_excel_converter.py

```diff
@@ -13,17 +13,24 @@
 from power_grid_model.data_types import SingleDataset
 
 from power_grid_model_io.converters import VisionExcelConverter
+from power_grid_model_io.data_stores.base_data_store import DICT_KEY_SUBNUMBER  # pylint: disable=unused-import
+from power_grid_model_io.data_stores.base_data_store import LANGUAGE_NL  # pylint: disable=unused-import
+from power_grid_model_io.data_stores.base_data_store import DICT_KEY_NUMBER, LANGUAGE_EN, VISION_EXCEL_LAN_DICT
 from power_grid_model_io.data_types import ExtraInfo
 from power_grid_model_io.utils.json import JsonEncoder
+from power_grid_model_io.utils.uuid_excel_cvtr import convert_guid_vision_excel
 
 from ..utils import compare_extra_info, component_attributes, component_objects, load_json_single_dataset, select_values
 
 DATA_PATH = Path(__file__).parents[2] / "data" / "vision"
 SOURCE_FILE = DATA_PATH / "vision_{language:s}.xlsx"
+SOURCE_FILE_97 = DATA_PATH / "vision_97_{language:s}.xlsx"
 VALIDATION_FILE = DATA_PATH / "pgm_input_data_{language:s}.json"
 LANGUAGES = ["en", "nl"]
+LANGUAGES_97 = ["en"]
```

Comment:
doesn't `nl` exist anymore? 😮

Reply:
I do not have a `nl` version test data yet